compound/src/main/kotlin/io/element/android/compound/theme/AvatarColors.kt

```diff
@@ -74,9 +54,9 @@ internal fun AvatarColorsLightPreview() {
 
 @Preview
 @Composable
-internal fun AvatarColorsDarkPreview() {
-    val chunks = avatarColorsDark.chunked(4)
-    ElementTheme {
+internal fun AvatarColorsPreviewDark() {
+    val chunks = avatarColors().chunked(4)
+    ElementTheme(darkTheme = true) {
```

Comment:
The `avatarColors()` should be invoked inside the `ElementTheme` block maybe?

Reply:
It should be fixed in https://github.com/element-hq/compound-android/pull/81/commits/91270a468da390acbb8b7b3e63a0bc927a1851f5. Nice catch!